File: app/src/main/java/com/cna/mineru/cna/DTO/ExamData.java
package com.cna.mineru.cna.DTO;

public class ExamData {
    public int Id;
    public int NoteId;
    public String Title;
    public int isSolved;
    public int RoomId;
    public String ExamTitle;
    public long TTS;

    public ExamData(int NoteId, String Title){
        this.NoteId = NoteId;
        this.Title = Title;
    }

    public ExamData(String ExamTitle, int RoomId){
        this.ExamTitle = ExamTitle;
        this.RoomId = RoomId;
    }

    public ExamData(int Id, int NoteId, String Title, long TTS, int isSolved){
        this.Id = Id;
        this.NoteId = NoteId;
        this.Title = Title;
        this.TTS = TTS;
        this.isSolved = isSolved;
    }
}

File: app/src/main/java/com/cna/mineru/cna/DTO/ExamData.kt
package com.cna.mineru.cna.DTO

class ExamData {
    var Id: Int = 0
    var NoteId: Int = 0
    var Title: String? = null
    var isSolved: Int = 0
    var RoomId: Int = 0
    var ExamTitle: String? = null
    var TTS: Long = 0

    constructor(TTS: Long, isSolved: Int) {
        this.TTS = TTS
        this.isSolved = isSolved
    }

    constructor(NoteId: Int, Title: String) {
        this.NoteId = NoteId
        this.Title = Title
    }

    constructor(ExamTitle: String, RoomId: Int) {
        this.ExamTitle = ExamTitle
        this.RoomId = RoomId
    }

    constructor(Id: Int, NoteId: Int, Title: String, TTS: Long, isSolved: Int) {
        this.Id = Id
        this.NoteId = NoteId
        this.Title = Title
        this.TTS = TTS
        this.isSolved = isSolved
    }
}
